cache-config/t3c-apply/torequest/torequest.go

```diff
@@ -1169,17 +1175,25 @@ func (r *TrafficOpsReq) UpdateTrafficOps(syncdsUpdate *UpdateStatus) error {
 
 	// TODO: The boolean flags/representation can be removed after ATC (v7.0+)
 	if !r.Cfg.ReportOnly && !r.Cfg.NoUnsetUpdateFlag {
+		start := time.Now()
+		apply := []string{}
+		var b bool
 		if r.Cfg.Files == t3cutil.ApplyFilesFlagAll {
-			b := false
+			b = false
+			apply = append(apply, "update")
+			log.Infof("Update flag currently set to %v, setting to %v", serverStatus.UpdatePending, b)
 			err = sendUpdate(r.Cfg, serverStatus.ConfigUpdateTime, nil, &b, nil)
 		} else if r.Cfg.Files == t3cutil.ApplyFilesFlagReval {
-			b := false
+			b = false
+			apply = append(apply, t3cutil.ApplyFilesFlagReval.String())
```

Comment:
This has the text value you want, but this enum is semantically something completely different.
The enum `ApplyFilesFlag` is the CLI flag for whether to apply all config files, or just the regex_revalidate.config file.
Whereas here, the word "reval" is being used to indicate revalidate pending flag in Traffic Ops.

That might confuse someone trying to understand the code in the future. Would you object to changing this to either a literal like update above, `"reval"`, or putting both in their own enum like `const TrafficOpsPendingFlag    TrafficOpsPendingFlagReval = "reval"`?